src/ontology/cl-edit.owl

```diff
@@ -19412,10 +19412,11 @@ SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0002603 obo:CL_0000127
 
 # Class: obo:CL_0002604 (astrocyte of the hippocampus)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-7258-9596"^^xsd:string) obo:IAO_0000115 obo:CL_0002604 "An astrocyte that is located in the hippocampus.")
```

Comment:
As mentioned offline, the following appears in the unreasoned file:
<img width="445" alt="Screenshot 2022-11-24 at 14 10 41" src="https://user-images.githubusercontent.com/95643797/203804644-9b4579b2-8e5e-464c-927b-3e20ab1b2d8d.png">

Reply:
Don't think its a problem, I think ORCIDS in this case are strings, not 100% sure, but anyway if its a problem down the road, its a simple SPARQL fix that should be applied to the whole ontology. QC should be up too if we need them as not strings